arborist/server.go

```diff
@@ -110,6 +115,7 @@ func (server *Server) MakeRouter(out io.Writer) http.Handler {
 	router.Handle("/policy/{policyID}", http.HandlerFunc(server.parseJSON(server.handlePolicyOverwrite))).Methods("PUT")
 	router.Handle("/policy/{policyID}", http.HandlerFunc(server.handlePolicyRead)).Methods("GET")
 	router.Handle("/policy/{policyID}", http.HandlerFunc(server.handlePolicyDelete)).Methods("DELETE")
+	router.Handle("/bulk/policy", http.HandlerFunc(server.parseJSON(server.handleBulkPoliciesOverwrite))).Methods("PUT")
```

Comment:
Can we use the Go http method consts? `http.MethodPut` for example

Reply:
what about consistency @williamhaley!?! 😆

Reply:
Really we should update all of them to use the `http` methods 🚀

Reply:
Sorry, I dont really know much about this. What would be the benefit of using the `http` methods and what is the downside of doing what we're doing right now? I'm guessing its just a standard to use the `http` methods? :thinking: